django/contrib/auth/tests/test_remote_user.py

```diff
@@ -17,16 +17,20 @@ class RemoteUserTest(TestCase):
     middleware = 'django.contrib.auth.middleware.RemoteUserMiddleware'
     backend = 'django.contrib.auth.backends.RemoteUserBackend'
     header = 'REMOTE_USER'
+    INITIAL_MIDDLEWARE_CLASSES = getattr(
+        settings, 'MIDDLEWARE_CLASSES', [])
+    INITIAL_AUTHENTICATION_BACKENDS = getattr(
+        settings, 'AUTHENTICATION_BACKENDS', [])
```

Comment:
I think this would be more readable if they were on one line each. It's fine if they are longer than 79 characters